docs/src/main/asciidoc/security-iap.adoc

```diff
@@ -12,9 +12,15 @@ The following claims are validated automatically:
  * Issue time
  * Expiration time
  * Issuer
+ * Audience
 
-Additionally, Audience (`"aud"`) validation is enabled when the application is running on AppEngine/ComputeEngine/Kubernetes, or when a custom audience is provided through `spring.cloud.gcp.security.iap.audience` property.
-The custom property overrides automatic environment-based checks.
+The audience (`"aud"`) validation is automatically configured when the application is running on AppEngine.
```

Comment:
```suggestion
The audience (`"aud"`) validation is automatically configured when the application is running on App Engine Standard.
```

Reply:
It should work the same on AppEngine Standard and AppEngine Flexible.